src/App.tsx

```diff
@@ -1,14 +1,126 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext, useEffect, useMemo, useRef, useState,
+} from 'react';
+import { addTodo, deleteTodo, getTodos } from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+// eslint-disable-next-line max-len
+import { ErrorNotification } from './components/ErrorNotification/ErrorNotification';
+import { Footer } from './components/Footer/Footer';
+import { Header } from './components/Header/Header';
+import { TodoInfo } from './components/TodoInfo/TodoInfo';
+import { TodoList } from './components/TodoList/TodoList';
+import { FilterTypes } from './types/Enums';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [filterType, setFilterType] = useState('All');
+  const [todoTitle, setTodoTitle] = useState('');
```

Comment:
I guess, we could move title state inside Header (cause it the filed of the form)(but its not necessary)

Reply:
agree, let's move everything related to the form of adding a todo to the header